manifests/v2.2.0/vsphere-67u3/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -102,10 +106,11 @@ spec:
       - name: liveness-probe
         image: quay.io/k8scsi/livenessprobe:v2.2.0
         args:
-        - --csi-address=/csi/csi.sock
+          - "--v=4"
+          - "--csi-address=/csi/csi.sock"
         volumeMounts:
-        - name: plugin-dir
-          mountPath: /csi
+          - name: socket-dir
```

Comment:
Can we make this name consistent across all containers. I guess node-driver-registrar has it as `plugin-dir`

Reply:
fixed it in the latest commit